server-kubernetes/cli/jgroups/standalone-ha-configuration.cli

```diff
@@ -0,0 +1,3 @@
+embed-server --server-config=standalone-ha.xml --std-out=echo
+run-batch --file=cli/jgroups/${env.JGROUPS_PROTOCOL}/change-jgroups.cli
```

Comment:
Good way to use the env var. I'm curious about the choice if env var name "_PROTOCOL". Is it more than the PING that needed changing? In my experience you only need to replace `tcp` `MPING` with something elsr.